packages/block-library/src/heading/edit.js

```diff
@@ -41,9 +39,9 @@ function HeadingEdit( { attributes, setAttributes, mergeBlocks, onReplace } ) {
 	return (
 		<>
 			<BlockControls>
-				<HeadingToolbar
-					minLevel={ Platform.OS === 'web' ? 2 : 1 }
-					maxLevel={ Platform.OS === 'web' ? 5 : 7 }
+				<HeadingLevelToolbar
+					minLevel={ 1 }
+					maxLevel={ 6 }
```

Comment:
Maybe we can drop the min/max prop all together?

Reply:
Yeah, now that I think about it, there's no reason to keep the props if there's only one unified toolbar.